package.yaml

```diff
@@ -19,10 +26,11 @@ executables:
       - http-client
       - lens
       - lens-aeson
+      - mtl
       - optparse-applicative
       - retry
       - text
       - unordered-containers
       - unix
       - utf8-string
-      - mtl
+      - vaultenv
```

Comment:
I would move all the dependencies here up to the new top-level dependencies section. And only keep `vaultenv` here. It's slightly nicer and also removes duplicates.

Reply:
Good suggestion, done!